src/SymfonyCache/RefreshListener.php

```diff
@@ -44,7 +44,7 @@ public static function getSubscribedEvents()
     public function handleRefresh(CacheEvent $event)
     {
         $request = $event->getRequest();
-        if (!$request->isMethodSafe()
+        if (!$request->isMethodSafe(false)
```

Comment:
looking at https://github.com/symfony/symfony/commit/34e7b956dd46cc03973ba90ce93523557a26a3e0 i think we misused this method to see if the request method is cacheable, so we should instead check if the method is cacheable. will have a look.

Reply:
Yeah, indeed I was unsure myself. Should've mentioned that, sorry!